components/pedigree/ui/src/main/resources/PhenoTips/PedigreeImageService.xml

```diff
@@ -37,13 +37,36 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{include reference="PhenoTips.PedigreeMacros" /}}
-
+  <content>
 {{velocity}}
+#if ("$!request.action" == 'export')
+  {{html wiki="false" clean="false"}}
+  $response.setContentType('application/json')##
+  #if ("$!request.image" == '')
+    $response.setStatus(404)##
+    {"error" : "$services.localization.render('phenotips.PedigreeImageService.noPedigreeImage')"}
+    #break
+  #end
+  ## Fix broken SVG generated by a broken version of the pedigree editor
+  #if ($stringtool.countMatches($request.image, 'xmlns=') &gt; 1)
```

Comment:
yeah. If it does not work and we have to ad new rules I would place it as a separate rule a few lines below where we do more replacings

btw you can always check in browser console whats the result of replacement to figure out at which point it fails